### PR TITLE
Add test for ui

### PR DESCRIPTION
**Pull request by seanjyjy**
Pr Overview
* Add ui test
* update ppp as well
* update previous test correctness

Note Ui Test has been removed in recent PRs due to unable to assert the fact that the test was done accurately.

### CODE REVIEW

File: src/test/java/seedu/internhunter/model/ModelManagerTest.java [hunk 1]
```diff
@@ -237,58 +237,58 @@ public void getFilteredProfileListSize_returnsEqual() {
...
+        assertEquals(profileItemItemList, modelManager.getUnfilteredProfileList());
     }
 
     @Test
     public void getUnfilteredApplicationList_equals_true() {
         modelManager.addApplication(applicationItemBuilder.build());
         ItemList<ApplicationItem> applicationItemItemList = new ItemList<>();
         applicationItemItemList.addItem(applicationItemBuilder.build());
-        assertEquals(modelManager.getFilteredApplicationList(), applicationItemList);
+        assertEquals(applicationItemList, modelManager.getFilteredApplicationList());
```
Comment (shawn-nyk) [inline src/test/java/seedu/internhunter/model/ModelManagerTest.java]: Should this be using `getUnfilteredApplicaitonList()` instead?

File: src/test/java/seedu/internhunter/ui/CardTest.java [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+            assertTrue(companyCard.equals(companyCard));
+            assertTrue(companyCardWithIndustries.equals(companyCardWithIndustries));
+
+            // null -> return false
+            assertFalse(companyCard.equals(null));
+            assertFalse(companyCardWithIndustries.equals(null));
+
+            // different type -> return false
+            assertFalse(companyCard.equals(0.5f));
+            assertFalse(companyCard.equals(0.5f));
```
Comment (shawn-nyk) [inline src/test/java/seedu/internhunter/ui/CardTest.java]: Should this be using `companyCardWithIndustries`?

**Review by shawn-nyk — CHANGES_REQUESTED** [review]
Thanks for the great work Sean! Just a few minor nits to fix and we should be good to merge.

File: src/test/java/seedu/internhunter/ui/CardTest.java (hunk 2 above)
Comment (seanjyjy) [reply src/test/java/seedu/internhunter/ui/CardTest.java]: good catch resolved

File: src/test/java/seedu/internhunter/model/ModelManagerTest.java (hunk 1 above)
Comment (seanjyjy) [reply src/test/java/seedu/internhunter/model/ModelManagerTest.java]: good catch resolved

**Review by shawn-nyk — APPROVED** [review]
LGTM

**Pull request merged**